From 6ea19d2610394e61f9a5367a3ce41c55ea01922d Mon Sep 17 00:00:00 2001
From: Igor Ulyanov <disinbox@gmail.com>
Date: Tue, 28 Jan 2014 11:48:13 +0400
Subject: [PATCH 2/3] remove getchar in gglobal

---
 src/lib/iglobal.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/lib/iglobal.c b/src/lib/iglobal.c
index f8c14b4..bb4d874 100644
--- a/src/lib/iglobal.c
+++ b/src/lib/iglobal.c
@@ -418,7 +418,7 @@ ttglobal gglobal()
 	if(iglobal == NULL)
 	{
 		printf("ouch - no state for this thread - hit a key to exit\n");
-		getchar();
+		//getchar();
 		//let it bomb exit(-1);
 	}
 	return iglobal;
-- 
1.8.5.1

